Mathlib/Algebra/AddTorsor/Basic.lean

```diff
@@ -148,18 +148,23 @@ namespace Pi
 universe u v w
 
 variable {I : Type u} {fg : I → Type v} [∀ i, AddGroup (fg i)] {fp : I → Type w}
+  [∀ i, AddTorsor (fg i) (fp i)]
 
 open AddAction AddTorsor
 
 /-- A product of `AddTorsor`s is an `AddTorsor`. -/
-instance instAddTorsor [∀ i, AddTorsor (fg i) (fp i)] : AddTorsor (∀ i, fg i) (∀ i, fp i) where
+instance instAddTorsor : AddTorsor (∀ i, fg i) (∀ i, fp i) where
   vadd g p i := g i +ᵥ p i
```

Comment:
I'm guessing this line can be removed, while you're here?

Reply:
No, the other `VAdd` instance is in `Mathlib.Algebra.Group.Action.Pi`, which is not imported by this file. Should I import that and remove the duplicate instance?

Reply:
Let's do that as a separate PR.